src/coreclr/System.Private.CoreLib/src/System/Threading/Interlocked.CoreCLR.cs

```diff
@@ -166,7 +168,6 @@ public static partial class Interlocked
         /// <exception cref="NullReferenceException">The address of <paramref name="location1"/> is a null pointer.</exception>
         /// <typeparam name="T">The type to be used for <paramref name="location1"/>, <paramref name="value"/>, and <paramref name="comparand"/>. This type must be a reference type.</typeparam>
         [return: NotNullIfNotNull("location1")]
-        [Intrinsic]
```

Comment:
This needs to be `[Intrinsics]` for getILIntrinsicImplementationForInterlocked

Reply:
Thanks! I assume other runtime side changes are ok?

Reply:
Yes, the other runtime side changes looks good. It would be nice to run jitdiffs to validate that this works as expected.

Reply:
yeah, I ran them after this changes and the PMI ones were 100% identical. For crossgen2 I'm having some unrelated issues atm and can't even compare Main against Main.